test/box/gh-5928-several-listen-sockets.test.lua

```diff
@@ -0,0 +1,246 @@
+test_run:cmd("create server test with script='box/gh-5928-several-listen-sockets.lua'")
+
+test_run:cmd("setopt delimiter ';'")
+function str_split (inputstr, sep)
+        local t = {}
+        if string.find(inputstr, sep) == nil then
+            t[#t + 1] = inputstr
+            return t
+        end
+        for str in string.gmatch(inputstr, "([^"..sep.."]+)") do
+                t[#t + 1] = str
+        end
+        return t
+end;
+function check_connection(server_addr)
+     local conn = net_box.new(server_addr)
+     if not conn then
+     	return false
+     end
+     local rc = true
+     if conn:ping() == false then
+         rc = false
+     end
+     conn:close()
+     return rc
+end;
+function prepare_several_listen_uri(default_server_addr, listen_sockets_count)
+    local listen = ""
+    for i = 1, listen_sockets_count do
+        local number = 97 + i - 1
+        local listen_addr = default_server_addr .. string.upper(string.char(number))
+        listen = listen .. listen_addr
+        if i ~= listen_sockets_count then
+            listen = listen .. ", "
+        end
+    end
+    return listen
+end;
+function check_connection_for_string_uri(server_addresses_str)
+    local connection_result = true
+    local server_addresses_table = str_split(server_addresses_str, ", ")
+    for i = 1, #server_addresses_table do
+        local t = str_split(server_addresses_table[i], "?")
+        if check_connection(t[1]) ~= true then
+            connection_result = false
+        end
+    end
+    return connection_result
+end;
+function check_connection_for_table_uri(server_addresses_table)
+    local connection_result = true
+    for _, server_addresses_str in pairs(server_addresses_table) do
+        connection_result = check_connection_for_string_uri(server_addresses_str)
+    end
+    return connection_result
+end;
+function check_connection_ex()
+    local server_addresses = test_run:eval('test', 'return box.cfg.listen')[1]
+    if type(server_addresses) == 'table' then
+        return check_connection_for_table_uri(server_addresses)
+    else
+        return check_connection_for_string_uri(server_addresses)
+    end
+end;
+function prepare_several_listen_uri(default_server_addr, listen_sockets_count)
+    local listen = ""
+    for i = 1, listen_sockets_count do
+        local number = 97 + i - 1
+        local listen_addr = default_server_addr .. string.upper(string.char(number))
+        listen = listen .. listen_addr
+        if i ~= listen_sockets_count then
+            listen = listen .. ", "
+        end
+    end
+    return listen
+end;
+test_run:cmd("setopt delimiter ''");
+
+test_run:cmd(string.format("start server test with args=\"%s\"", 1))
+default_server_addr = test_run:eval('test', 'return box.cfg.listen')[1]
+test_run:cmd("stop server test")
+
+-- Checks that we able to open several listening sockets in several threads
+-- Checks that all unix socket pass successfully deleted after stop listening
```

Comment:
Nit: socket pass --> socket path I guess..?
Nit: after stop listening --> after listening is stop

Reply:
Fixed